content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +157,50 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
+
+To control which sysctls can be set in pods, specify the
+`forbiddenSysctls` and/or `allowedUnsafeSysctls` fields in the PodSecurityPolicy.
+
+By default, all safe sysctls in the whitelist are allowed.
+
+Both `forbiddenSysctls` and `allowedUnsafeSysctls` are lists of plain sysctl names
+or sysctl patterns (which end with `*`). The string `*` matches all sysctls.
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The `forbiddenSysctls` field excludes specific sysctls, and can include a
```

Comment:
nit: excludes vs. include is confusing. Better use a different word for "include".